src/message/persist.rs

```diff
@@ -614,6 +614,7 @@ impl<'r> From<&'r super::BalanceMessage> for models::BalanceHistory {
             balance_available: DecimalDbType::from_str(&origin.balance_available).unwrap_or_else(decimal_warning),
             balance_frozen: DecimalDbType::from_str(&origin.balance_frozen).unwrap_or_else(decimal_warning),
             detail: origin.detail.clone(),
+            signature: "".to_owned(),
```

Comment:
why `signature: origin.signature.as_bytes().to_vec(),` for `impl<'r> From<&'r super::TransferMessage> for models::InternalTx {`

but `signature: "".to_owned(),` for `impl<'r> From<&'r super::BalanceMessage> for models::BalanceHistory {`?

Reply:
Yes, it is wrong. I will fix it. Thanks.